graph/src/data/store/mod.rs

```diff
@@ -246,6 +247,7 @@ impl Value {
             (r::Value::Int(i), _) => Value::Int(*i as i32),
             (r::Value::Boolean(b), _) => Value::Bool(b.to_owned()),
             (r::Value::Null, _) => Value::Null,
+            // Kamil: we should support r::Value::Object here if we want to add #2960
             _ => {
                 return Err(QueryExecutionError::AttributeTypeError(
```

Comment:
@lutter  but we can't convert `Value::Object` to `r::Value` because it's not supported